### PR TITLE
Build: Reset build number

### PR DESCRIPTION
**Pull request by dbast**
There is no defaults release yet, so reseting the build number to get it built and uploaded.

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
Actions:
1. Skip `py<37`
2. Fix python in `host` and `run`
3. Add `conda_package_streaming` to `imports`
4. Add `license_family`

**Comment by skupr-anaconda** [comment]
The Prefect `linux-s390x` agent isn't working (status: queued)

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,33 +10,35 @@ source:
...
-    - pip check
   requires:
     - pip
     - requests
+  commands:
+    - pip check
 
 about:
   home: https://github.com/conda/conda-package-streaming
   summary: Download metadata from conda packages without transferring entire file.
```
Comment (dholth) [inline recipe/meta.yaml]:
```suggestion
  summary: An efficient library to read from new and old format .conda and .tar.bz2 conda packages.
```

**Review by dholth — APPROVED** [review]
Thanks for building my little project!

**Comment by dholth** [comment]
Let's merge some of these changes (not the ones making it not-noarch) into https://github.com/conda/conda-package-streaming/blob/main/conda.recipe/meta.yaml

**Review by katietz — APPROVED** [review]
lgtm.  no description for it?

**Comment by dbast** [comment]
@katietz description: this is an important/required dependency of conda-package-handling 2.0.

**Comment by dholth** [comment]
The linter thinks `wheel` should not be required; but since `conda-package-streaming` uses `flit-core` IIUC the `wheel` package is not used at all.

**Comment by skupr-anaconda** [comment]
Another package implicitly uses wheel in `host` https://cloud.prefect.io/anaconda/flow-run/3b6767a7-ef45-4e1f-afc8-9d109c8db809?logs=&logId=2aef5603-c4dc-4a51-9db0-b38b3edcee81

**Comment by dholth** [comment]
I see. Sounds like transitive dependencies work.

```
06:47:33
INFO
build
    wheel:            0.37.1-pyhd3eb1b0_0      defaults

**Pull request merged**